packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -12,9 +12,6 @@ export const ALIAS = 'alias';
 export const ANSWER = 'answer';
 
 export const CALL = 'call';
-export const CORRELATION_ID = 'correlationId';
-export const MEETINGNUMBER = 'meetingNumber';
-export const CONVERSATION_URL = 'conversationUrl';
```

Comment:
It still works if they use 'correlationId' but for CORRELATION_ID const i remove as per below Marcin's comments

Marcin Wojtczak: yeah it will be a breaking change but it will be easy for any client to update and use the correct value from the new enum and if they just use hardcoded strings and JS (not TS) then it will still work fine if we keep the same values in the enum as the old consts. Also I think majority of other SDK clients are still not on the 3.x version right? and 3.x introduces lots of breaking API changes so now is a good time to do any other API breaking changes (it was even better time when we were still on beta branch, but now is still better than later)

Reply:
@rsarika - 

> then it will still work fine if we keep the same values in the enum as the old consts

This is what I am concerned about. In your change, the enum doesn't keep the old values. The old values go away.

Reply:
My bad. I think I was confused between the two constants `MEETING_KEY` and `DESTINATION_TYPE`

You were right to remove the old constants. We'll discuss this in person on what confused me. Sorry about this @rsarika